### PR TITLE
Remove File.separator in Paths.get()

### LINKED ISSUE #69: File.separator is not needed in Paths.get()
Since `Paths.get()` returns automatically the resulting path by combining the strings supplied to it as arguments with the system-dependent default file separator , there is no need to use the ` File.separator` explicitly.

### PR DESCRIPTION
**Pull request by kasunsiyambalapitiya**
Resolve #69

### CODE REVIEW

File: carbon-feature-plugin/src/main/java/org/wso2/maven/p2/feature/install/FeatureInstaller.java [hunk 1]
```diff
@@ -153,7 +153,7 @@ private void deleteOldProfiles() throws IOException {
     private void writeEclipseIni() throws IOException {
         String profileLocation = resourceBundle.getDestination() + File.separator + resourceBundle.getProfile();
 
-        File eclipseIni = Paths.get(profileLocation + File.separator + "eclipse.ini").toFile();
+        File eclipseIni = Paths.get(profileLocation , "eclipse.ini").toFile();
```
Comment (Niranjan-K) [inline carbon-feature-plugin/src/main/java/org/wso2/maven/p2/feature/install/FeatureInstaller.java]: need to code format. extra space after "profileLocation"

File: carbon-feature-plugin/src/main/java/org/wso2/maven/p2/utils/FileManagementUtil.java [hunk 2]
```diff
@@ -48,8 +48,7 @@
      * @return the config.ini file for the Profile
      */
     public static File getProfileConfigIniFile(String destination, String profile) {
-        return Paths.get(destination + File.separator + profile + File.separator + "configuration" +
-                File.separator + "config.ini").toFile();
+        return Paths.get(destination , profile , "configuration" , "config.ini").toFile();
```
Comment (Niranjan-K) [inline carbon-feature-plugin/src/main/java/org/wso2/maven/p2/utils/FileManagementUtil.java]: same as above

**Review by Niranjan-K — CHANGES_REQUESTED** [review]

File: carbon-feature-plugin/src/main/java/org/wso2/maven/p2/feature/install/FeatureInstaller.java (hunk 1 above)
Comment (kasunsiyambalapitiya) [reply carbon-feature-plugin/src/main/java/org/wso2/maven/p2/feature/install/FeatureInstaller.java]: fixed in 1b68b28 commit

File: carbon-feature-plugin/src/main/java/org/wso2/maven/p2/utils/FileManagementUtil.java (hunk 2 above)
Comment (kasunsiyambalapitiya) [reply carbon-feature-plugin/src/main/java/org/wso2/maven/p2/utils/FileManagementUtil.java]: fixed in 1b68b28 commit

**Review by Niranjan-K — APPROVED** [review]

**Pull request merged**